docs/page-api.md

```diff
@@ -10,7 +10,7 @@ A binding of URL paths to a Page class that can render the results for those URL
 
 * The HTTP method(s) that are acceptable.
 
-* Default: “get”
+* Default: [“get”, "head"]
```

Comment:
Not strictly necessary and if people disagree, it can easily be changed back.  I like to have parity between GET/HEAD in most cases.